pkgs/by-name/ki/kittysay/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}: let
+  version = "0.5.0";
+in
+  rustPlatform.buildRustPackage {
+    pname = "kittysay";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "uncenter";
+      repo = "kittysay";
+      rev = "v${version}";
+      sha256 = "sha256-eOcHrEvU3nBKFokwE8CyDOUYoBA1+gBlnl7VRUuoFfE=";
+    };
+
+    cargoHash = "sha256-dVgPp5jY3ii8mO/HLTDESQzQyZXzqut8Bjm2KfWD0+U=";
+
+    meta = {
+      description = "cowsay, but with a cute kitty :3";
```

Comment:
```suggestion
      description = "Cowsay, but with a cute kitty :3";
```

Reply:
Technically this isn't how the program cowsay is called? It's meant to be spelled lowercase.

Reply:
seems to be in lowercase on wikipedia too. I'll revert this change.

Reply:
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.

Reply:
Capitalizing the first character makes sense in general but this sentence in particular begins with the name of a package?

Reply:
I don't think that's a valid exception here.